src/components/blog-bio/bio.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Blog Bio component
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import {GatsbyImage} from 'gatsby-plugin-image';
+
+import ShortBio from './short-bio';
+import * as styles from './blog-bio.module.scss';
+import PersonSocialMediaLinks from '../person/social-media-links';
+
+const Bio = ({fullName, writtenDate, avatar, email, github, linkedIn}) => {
+  return (
+    <div className={styles.bioContainer}>
+      <div className={styles.avatarColumn}>
+        <GatsbyImage
+          className={styles.avatarImg}
+          alt={fullName || 'Author Picture'}
+          image={avatar}
+        />
+      </div>
+      <div>
+        <p>
+          <ShortBio fullName={fullName}/>
```

Comment:
Thoughts on adding job name here also?

Reply:
I think go with the design of Gianluca on this one.